### PR TITLE
Fix 424 Failed Dependency

### PR DESCRIPTION
**Pull request by mikecat**
### Description

Fix an issue that the "Status" section had the specification.

### CODE REVIEW

File: files/en-us/web/http/status/424/index.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Regular web servers will normally not return this status code. But some other pr
 
 ## Status
 
+```plain
```
Comment (jasonren0403) [inline files/en-us/web/http/status/424/index.md:16]: we may use http highlight here as other status document does

```suggestion
```http
```

**Review by hamishwillee — APPROVED** [review]
Thank you. It is good to have consistent docs.

**Comment by jasonren0403** [comment]
hi @hamishwillee , this PR had got an approval, but havent' been merged yet. Is there still anything blocking?

**Comment by hamishwillee** [comment]
No - I just forgot sorry!

**Pull request merged**